### PR TITLE
Warn that Krun can't check for turbo mode on OpenBSD.

### PR DESCRIPTION
**Pull request by ltratt**
Fixes https://github.com/softdevteam/krun/issues/340

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -185,6 +185,11 @@ Commonly enabled services that you may wish to disable:
  * pflogd
  * sndiod
 
+Note that Krun is unable to check whether turbo mode is enabled on OpenBSD or
+not, and is also unable to use APERF/MPERF ratios to indirectly check whether
```
Comment (vext01) [inline README.md]: Kill "or not"?

**Comment by vext01** [comment]
This concludes my review.

File: README.md (hunk 1 above)
Comment (ltratt) [reply README.md]: I don't think the sentence parses without "or not" ("whether" requires two clauses).

Comment (vext01) [reply README.md]: Both are fine for me, so as you wish.

Comment (ltratt) [reply README.md]: Let's keep it as-is. I think we can then merge?

**Pull request merged**